src/login/Login.h

```diff
@@ -41,19 +41,283 @@ struct ProfileRecord
 	std::string hotkey;
 	std::string characterClass;
 	int characterLevel = 0;
+
+	std::optional<std::string> eqPath;
 	bool checked = false;
 
+	std::optional<bool> endAfterSelect;
+	std::optional<int> charSelectDelay;
+	std::optional<std::filesystem::path> customClientIni;
```

Comment:
this could probably be a string. I think most places i found use it as a string anyways.